libdnf/sack/query.cpp

```diff
@@ -1340,7 +1341,7 @@ Query::Impl::filterAdvisory(const Filter & f, Map *m, int keyname)
         id = resultPset->next(id);
         if (id == -1)
             break;
-        Solvable* s = pool_id2solvable(pool, id);
+        _Solvable* s = pool_id2solvable(pool, id);
```

Comment:
Please what the sense to do that? Solvable* to _Solvable*.

Reply:
Solvable != _Solvable, see #411